### ISSUE TITLE
BingGPT no longer responding to my messages

### ISSUE THREAD

**Issue by AutonomousCat** [dice2o/BingGPT#163]
![image](https://user-images.githubusercontent.com/130127508/234251647-63c30937-d518-4299-9a57-8139214ef305.png)

**Comment by jidehem92** [comment]
Same problem for me. Compose is still functionnal

**Comment by a77lic7ion** [comment]
me too

**Comment by Dqharden** [comment]
me too

**Comment by sir-wallaby** [comment]
Yes, same issue here using [BingGPT-0.3.3-linux-x64.deb](https://github.com/dice2o/BingGPT/releases/download/v0.3.3/BingGPT-0.3.3-linux-x64.deb)

**Comment by lyzergik** [comment]
Same here

**Comment by laurencejbelliott** [comment]
I'm having this same issue using [BingGPT-0.3.3-darwin-arm64.dmg](https://github.com/dice2o/BingGPT/releases/download/v0.3.3/BingGPT-0.3.3-darwin-arm64.dmg)

**Comment by Bono2007** [comment]
Same issue with arm64 version

**Comment by yangfengwei** [comment]
I can't even open it today.

**Comment by AlexMnrs** [comment]
Same here, Windows 11 22624.1616. Now you can't even see the Chat nor Compose tabs. It's full gray.

![image](https://user-images.githubusercontent.com/10886760/234390818-c0a67031-67fa-4ae9-b814-be3468acfbc1.png)

**Comment by 5a5ch4** [comment]
It doesn't work for me anymore either.
<img width="519" alt="image" src="https://user-images.githubusercontent.com/38190757/234415580-0941190d-1a0e-4143-9fc5-ce22700e30d4.png">

**Comment by Arthur2012** [comment]
https://edgeservices.bing.com/
<h2>Our services aren't available right now</h2><p>We're working to restore all services as soon as possible. Please check back soon.</p>

**Comment by laurencejbelliott** [comment]
It's working for me again now. Just an outage on Microsoft's end it seems. Perhaps this issue can be closed if it's working again for others @dice2o ?
